senderworker/scripts/create-local-conf-from-CODE.sh

```diff
@@ -0,0 +1,56 @@
+function readStringParameter() {
```

Comment:
This is a really useful function! It might be worth moving into a common/shared location because it's likely this would be useful for locally running the lambdas too

Reply:
It is great that it can be useful!  I move the script folder up a level to be directly under `mobile-n10n` directory